AddressService/AddressService.AzureFunction/Startup.cs

```diff
@@ -110,6 +114,7 @@ public override void Configure(IFunctionsHostBuilder builder)
 
             builder.Services.AddMediatR(typeof(GetPostcodeHandler).Assembly);
             builder.Services.AddMediatR(typeof(GetNearbyPostcodesHandler).Assembly);
```

Comment:
Don't think you need to add one per handler. The .Assembly should take of all mediators within a project

Reply:
true